schemas/context/consentstring.schema.json

```diff
@@ -0,0 +1,60 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/consentstring",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Consent String",
+  "type": "object",
+  "meta:extensible": true,
+  "description":
+    "Consent String as presented by user along with standard information to help consent enforcer to parse the String.",
+  "definitions": {
+    "consentstring": {
+      "properties": {
+        "xdm:consentStandard": {
+          "title": "Standard for the consent",
+          "type": "string",
+          "description":
+            "This helps determine the format of the consent string as set by CMP."
```

Comment:
What is CMP?  Spell it out...

Reply:
Updated to generic consent management service